global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.400",
+    "version": "7.*",
```

Comment:
Please revert this. We stick to LTS versions.

Reply:
Why cannot we use the `DotPulsar`? It supports .NET Standard 2.0 and above. It should be enough to test the container configuration and does not require so many changes.